client/src/pages/EquipmentBrowser/EquipmentBrowser.module.css

```diff
@@ -33,28 +27,85 @@
   background: rgb(171, 172, 231);
   justify-content: space-between;
   max-width: 65%;
-  flex-flow: row wrap;
 }
 
 .equipmentGrid > * {
   flex-flow: row-wrap;
   align-items: center;
 }
 
-.selectedEquipment {
+.cart {
   display: flex;
-  background: rgb(176, 177, 231);
+  background: white;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 12px;
+  color: #628aba;
+  justify-content: center;
   flex-grow: 1;
   flex-basis: 30%;
 }
 
 .collapsehead {
-  background: #add8e6;
-  margin-bottom: 10px;
-  border: 1px solid #0067cd;
+  background: #3279d5;
+  font-family: Roboto;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 12px;
+  padding: 6px;
+  color: white;
   cursor: pointer;
+  margin: 11px;
 }
```

Comment:
```suggestion
  font-size: 18px;
  padding: 6px 10px 0;
  color: white;
  cursor: pointer;
  margin: 11px;
}

/* the "trigger" section of the Collapsible */
.collapsehead :global(.Collapsible__trigger) {
  display: flex;
  justify-content: space-between;
  align-items: center;
  padding-bottom: 6px;
}

.collapsehead :global(.Collapsible__trigger) svg {
  transition: transform 0.25s;
}

/* Animate flipping the icon when the Collapsible is open */
.collapsehead :global(.Collapsible__trigger.is-open) svg {
  transform: rotate(-180deg);
}

/* Add some space between the bottom of the content and the bottom of the container so that the blue border goes all the way around (this is a nitpicky detail) */
.collapsehead :global(.Collapsible__contentInner) { margin-bottom: 10px; }
```